test/e2e/kubectl.go

```diff
@@ -907,7 +930,8 @@ var _ = framework.KubeDescribe("Kubectl client", func() {
 			framework.WaitForRCToStabilize(c, ns, rcName, framework.PodStartTimeout)
 
 			By("rolling-update to same image controller")
-			framework.RunKubectlOrDie("rolling-update", rcName, "--update-period=1s", "--image="+nginxImage, "--image-pull-policy="+string(api.PullIfNotPresent), nsFlag)
+
+			runKubectlOrDieRetryOptimisticLock("rolling-update", rcName, "--update-period=1s", "--image="+nginxImage, "--image-pull-policy="+string(api.PullIfNotPresent), nsFlag)
```

Comment:
Suggest a more general name (we may want to retry on other errors in the future?), e.g. `runKubectlRetryOrDie`